release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1846,34 +1855,16 @@ In the following tables, features are marked with the following statuses:
 |oc-mirror plugin v2
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability 
 
 |oc-mirror plugin v2 enclave support
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
 
 |oc-mirror plugin v2 delete functionality
 |Technology Preview
 |Technology Preview
-|Technology Preview
-
-|====
-
-[discrete]
-[id="ocp-release-notes-extensions-tech-preview_{context}"]
-=== Extensions Technology Preview features
-
-// "Extensions" refers to OLMv1
-
-.Extensions Technology Preview tracker
-[cols="4,1,1,1",options="header"]
-|====
-|Feature |4.16 |4.17 |4.18
-
-|{olmv1-first}
-|Technology Preview
-|Technology Preview
```

Comment:
I think this might have been accidentally deleted somehow, please add these lines back in (and still ensure that the delete functionality entry shows GA for 4.18)